### PR TITLE
selenium: Add test_loginpage.py

### PR DESCRIPTION
**Pull request by kishansag**
Test retrieval of loginpage  as well as enters email and password in the form"

### CODE REVIEW

**Comment by rajat1994** [comment]
@kishansag .... Did you install selenium locally and tried to run this test (py.test tests/e2e/)?

**Comment by kishansag** [comment]
I have selenium installed and have ran tests locally, works perfectly fine. I don't know why it shows error.

**Comment by rajat1994** [comment]
@kishansag ... travis build is showing this error   ... 
Unable to locate element: {"method":"id","selector":"email"}

**Comment by krzysztof** [comment]
@kishansag Take a look at this PR by @rajat1994 https://github.com/zenodo/zenodo/pull/994

**Comment by kishansag** [comment]
@krzysztof I have made some changes. Can you review this?

**Comment by krzysztof** [comment]
Hey, you're opening the page properly, however this test should:

1. Use an existing user credentials and try to actually log in. Just add a user fixture here: https://github.com/zenodo/zenodo/blob/master/tests/e2e/conftest.py
Similar to the users fixture from unit: https://github.com/zenodo/zenodo/blob/master/tests/unit/conftest.py#L206-L226

1. Make some assertions, otherwise this is not really testing anything. E.g. this test displays an error like this:
sc

(You can take screenshots by calling: ``env_browser.save_screenshot('/path/to/file.png')``)

File: tests/e2e/test_loginpage.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+# Ignore SIG_PIPE and don't throw exceptions on it...
+# (http://docs.python.org/library/signal.html)
+signal(SIGPIPE, SIG_DFL)
+
+
+def test_loginpage(live_server, env_browser, users):
+    """Test retrieval of loginpage"""
+    """ as well as enters email and password in the form"""
+    env_browser.get(url_for('security.login', _external=True))
+    email = users[0].email
```
Comment (krzysztof) [inline tests/e2e/test_loginpage.py]: users[0]['email'] - it's a dictionary.

File: tests/e2e/test_loginpage.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    password = users[0].password
+    elem = env_browser.find_element_by_id("email")
+    elem.send_keys(email)
+    elem = env_browser.find_element_by_id("password")
+    elem.send_keys(password)
+    elem.send_keys(Keys.RETURN)
```
Comment (krzysztof) [inline tests/e2e/test_loginpage.py]: You need to add some assertions in this test.

File: tests/e2e/test_registerpage.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+def test_registerpage(live_server, env_browser):
+    """Test retrieval of registerpage."""
+    env_browser.get(
+        url_for('security.register', _external=True))
+    email = 'info@zenodo.org'
+    username = 'info'
+    password = 'tester'
+    elem = env_browser.find_element_by_id("email")
+    elem.send_keys(email)
+    elem = env_browser.find_element_by_id("profile.username")
```
Comment (krzysztof) [inline tests/e2e/test_registerpage.py]: only username or email is required for logging in

**Comment by lnielsen** [comment]
Closing in favour of https://github.com/inveniosoftware/invenio-app-ils/commit/28d6e6baacf166b619fc0bf1b139334c7c2a5c3c

**Pull request closed without merging**